src/models/mod.rs

```diff
@@ -6,23 +6,74 @@ mod layout;
 mod link;
 mod metadata;
 mod predicate;
-mod state;
+mod statement;
 
 pub use helpers::*;
 pub use layout::*;
 pub use link::*;
 pub use metadata::*;
-pub use predicate::{PredicateLayout, PredicateVersion, PredicateWrapper};
-pub use state::{StateLayout, StateWrapper, StatementVer};
+pub use predicate::{PredicateLayout, PredicateVer, PredicateWrapper};
+pub use statement::{StatementLayout, StatementVer, StatementWrapper};
 
 #[cfg(test)]
 mod test {
+    use std::collections::{BTreeMap, HashMap};
+
+    use data_encoding::HEXLOWER;
     use once_cell::sync::Lazy;
 
-    use super::{LinkMetadata, LinkMetadataBuilder};
+    use crate::crypto::{HashAlgorithm, HashValue};
+
+    use super::{
+        link::metadata::MetadataFlatten, LinkMetadataBuilder, TargetDescription, VirtualTargetPath,
+    };
+
+    pub static TEST_MATERIALS: Lazy<BTreeMap<VirtualTargetPath, TargetDescription>> =
+        Lazy::new(|| {
+            BTreeMap::from([
+                (
+                    VirtualTargetPath::new(
+                        "git+https://github.com/curl/curl-docker@master".to_owned(),
+                    )
+                    .unwrap(),
+                    HashMap::from([(
+                        HashAlgorithm::Sha256,
+                        HashValue::new(
+                            HEXLOWER
+                                .decode("d6525c840a62b398424a78d792f457477135d0cf".as_bytes())
+                                .unwrap(),
+                        ),
+                    )]),
+                ),
+                (
+                    VirtualTargetPath::new("github_hosted_vm:ubuntu-18.04:20210123.1".to_owned())
+                        .unwrap(),
+                    HashMap::new(),
+                ),
+            ])
+        });
 
-    pub static BLANK_META: Lazy<LinkMetadata> = Lazy::new(|| {
-        let builder = LinkMetadataBuilder::default();
+    pub static TEST_FLATTEN: Lazy<MetadataFlatten> = Lazy::new(|| {
```

Comment:
Shared test data source to generated server attestation from same `MetadataFlatten`.